bundles/core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v1/aemform/AEMFormImpl.java

```diff
@@ -55,15 +56,19 @@ public class AEMFormImpl extends AbstractComponentImpl implements AEMForm {
     private static final Logger LOGGER = LoggerFactory.getLogger(AEMFormImpl.class);
 
     @ScriptVariable
+    @Optional
     protected PageManager pageManager;
 
     @ScriptVariable
+    @Optional
     private Page currentPage;
 
     @ScriptVariable
+    @Optional
     protected Style currentStyle;
 
     @ScriptVariable
+    @Optional
```

Comment:
remove this

Reply:
Removed

Reply:
Can not remove this as this is throwing error because in our case it needs to be optional.